config/optional/block.block.thunder_admin_help.yml

```diff
@@ -0,0 +1,15 @@
+langcode: en
+status: true
+dependencies: { }
```

Comment:
We need a dependency on the help module here. Otherwise this gets installed even without the help module enabled, which leads to "This block is broken or missing. You may be missing content or you might need to enable the original module. " Error messages.